word_usage.py

```diff
@@ -10,16 +10,46 @@
 max_frequency = 0
 words_only = False
 no_repeats = False
+split_apostrophe = False
+
+
+def clean(line, split=False):
+    # Remove apostrophes to split words cutted
+    line = line.replace("'", " ")
+
+    # Convert curly apostrophes to straight
+    line = line.replace(u"\u2018", "'")
+    line = line.replace(u"\u2019", "'")
+    line = line.replace(u"\u0060", "'")
+    line = line.replace(u"\u00b4", "'")
+
+    # Filter out symbols
+    line = re.sub("[^a-zA-Z\u00c0-\u024f\u1e00-\u1eff']", " ", line)
+
+    # Remove any double spaces introduced by last regex
+    line = re.sub("\ {2,}", " ", line)
+
+    line = line.lower()
+
+    if split:
+        return line.split()
+
+    return line
+
+
+def printhelp():
+    print('word_usage.py -i <input file> [-d <dictionary>] [--limit x] [--min-frequency x] [--max-frequency x] [--show-words-only] [--strip-by-apostrophe] [--no-repeats]')
+
 
 try:
-    opts, args = getopt.getopt(sys.argv[1:],"i:d",["input=","dictionary=","limit=","min-frequency=","max-frequency=","show-words-only","no-repeats"])
+    opts, args = getopt.getopt(sys.argv[1:],"i:d",["input=","dictionary=","limit=","min-frequency=","max-frequency=","show-words-only","strip-by-apostrophe","no-repeats"])
```

Comment:
The variable is called `split_apostrophe` while the parameter is `strip-by-apostrophe`. I think `split-by-apostrophe` or `split-apostrophe` is a better description of what the parameter does.